app/sliders/importer.py

```diff
@@ -166,6 +174,48 @@ def init_nested_dict(self):
         :return: defaultdict, a nested dictionary
         """
         return defaultdict(self.init_nested_dict)
+    
+
+class FUEDataParser:
+    def parse(self, sheets):
+        data = self.extract(sheets)
+
+        return data
+    
+    def extract(self, sheets):
+        """
+        Extract Fue Expenses data.
+
+        :param sheets: dict, where keys are names of sheets and values are pandas DataFrames
+        :return: nested dict, with the following shape {area: {year: {x: unit[], y: value[]}}}
+        """
+        extracted = self.init_nested_dict()
+        relevant_years = [2011, 2020]
+        for sheet_key in sheets:
+            basis, _, area = sheet_key.split("_")
+            if basis == "fue":
```

Comment:
Same indication I made for the basis parser. We can do the check a line higher:

for sheet_key in sheets:
    if "fue_ausgaben" in sheet_key:
          fue, ausgaben, area = sheet_key.split("_")